FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -131,6 +140,28 @@ extension SetupInitialViewController {
         handleCommon(error: error)
         state = .error(error)
     }
+
+    private func checkClientConfigurationIntegrity() {
+        guard let integrityErrorMessage = clientConfigurationService.checkForUsingKeyManager() else {
+            state = .checkingEKMKeys
+            return
+        }
+        showAlert(message: integrityErrorMessage) { [weak self] in
+            self?.router.signOut()
+        }
+    }
+
+    private func checkEKMKeys() {
+        organisationalRulesService.getEmailKeyManagerPrivateKeys()
+            .then { [weak self] result in
+                if result.keys.isNotEmpty {
```

Comment:
I just changed my mind seeing this - please remove this `if` here, and show the toast always, as long as we got a result from EKM. Even if it's zero keys.